experimental/traffic-portal/src/app/core/statuses/status-details/status-details.component.scss

```diff
@@ -0,0 +1,12 @@
+mat-card {
```

Comment:
This file is missing a license header. See other stylesheets for a comment block at the top that you can copy.